arbos/tx_processor.go

```diff
@@ -628,8 +627,15 @@ func (p *TxProcessor) L1BlockHash(blockCtx vm.BlockContext, l1BlockNumber uint64
 	return hash, nil
 }
 
+func (p *TxProcessor) DropTip() bool {
+	return p.state.ArbOSVersion() < 8 || p.delayedInbox
```

Comment:
We'll want to make this ArbOS version 9 now that we're planning an intermediate release

Reply:
Bumped!